sql/step_04_all_following_steps.sql

```diff
@@ -105,3 +105,4 @@ source step_a108_audit_dicom_type_and_dicom_data.sql;
 source step_a109_update_user_settings_table_constraints_and_add_usergroups_permissions.sql;
 source step_a110_update_operations_table_change_ope_for_to_enum.sql;
 source step_a111_add_missing_lock_columns.sql;
+source step_a112_add_medical_ward_lock_column.sql;
```

Comment:
Sorry, noticed now, can you use the already existing `step_a111` for this purpose?
Strange it is not conflicting, in CORE we already have this situation:

```
...
source step_a109_update_user_settings_table_constraints_and_add_usergroups_permissions.sql;
source step_a110_update_operations_table_change_ope_for_to_enum.sql;
source step_a111_add_missing_lock_columns.sql;
source step_a112_users_and_groups_soft_deletion.sql;

```

Reply:
OK !